editor/plugins/script_editor_plugin.cpp

```diff
@@ -3004,6 +3018,14 @@ void ScriptEditor::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("script_close", PropertyInfo(Variant::OBJECT, "script", PROPERTY_HINT_RESOURCE_TYPE, "Script")));
 }
 
+void ScriptEditor::_filter_scripts_text_changed(const String &p_newtext) {
```

Comment:
Also, as those arguments aren't read, they don't need to be named (this also applies to the ones in the header).

Reply:
I don't understand what "don't need to be named" means. What should I do?

Reply:
Would be something like this `_filter_scripts_text_changed(const String)`, without the named param. Both in the ".cpp" and ".h" files.

It's removed by some functions on the code, but it's also left in in others, so you decide if it's worthy.

Reply:
I think it should be kept as is, it makes it clearer that the signal does pass an argument with the new text. It can then be used in the future if needed.

Reply:
left it as it is